docs/scripts/reference.py

```diff
@@ -95,6 +95,9 @@ def gen_reference_api():
                 or _is_error_cls(obj)
                 or _is_tag_formatter_instance(obj)
                 or _is_tag_formatter_cls(obj)
+                or _is_extension_command_api(obj)
```

Comment:
This is added so that the entries related to extension hooks, urls, and commands are excluded from the main reference page.